### PR TITLE
test/utils: move 'ExecuteCommandOnNodeThroughVirtHandler' to libnode

### PR DESCRIPTION
**Pull request by oshoval**
### What this PR does
Move it to libnode, since it is related to node and already uses a libnode function.

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
None
```

### CODE REVIEW

**Review by ormergi — APPROVED** [review]
Looks good

Please consider changing the PR title its confusing, maybe something along this line "test/utils: move 'ExecuteCommandOnNodeThroughVirtHandler' to libnode"

**Comment by oshoval** [comment]
> Looks good
> 
> Please consider changing the PR title its confusing, maybe something along this line "test/utils: move 'ExecuteCommandOnNodeThroughVirtHandler' to libnode"

Done

File: tests/libnode/node.go [hunk 1]
```diff
@@ -443,3 +445,14 @@ func GetNodeHostModel(node *k8sv1.Node) (hostModel string) {
 	}
 	return hostModel
 }
+
+func ExecuteCommandOnNodeThroughVirtHandler(
+	virtCli kubecli.KubevirtClient,
```
Comment (codingben) [inline tests/libnode/node.go:450]: There is no need to pass `kubecli.KubevirtClient` as you can use `kubevirt.Client()` directly, it's a singleton.

File: tests/swap_test.go [hunk 2]
```diff
@@ -224,7 +224,7 @@ var _ = Describe("[Serial][sig-compute]SwapTest", Serial, decorators.SigCompute,
 })
 
 func getMemInfoByString(node v1.Node, field string) int {
-	stdout, stderr, err := tests.ExecuteCommandOnNodeThroughVirtHandler(kubevirt.Client(), node.Name, []string{"grep", field, "/proc/meminfo"})
+	stdout, stderr, err := libnode.ExecuteCommandOnNodeThroughVirtHandler(kubevirt.Client(), node.Name, []string{"grep", field, "/proc/meminfo"})
```
Comment (codingben) [inline tests/swap_test.go:227]: `kubevirt.Client()` can be used in the function itself.

**Review by codingben — CHANGES_REQUESTED** [review]
Please use `kubevirt.Client()` directly and I'll add lgtm label.

File: tests/swap_test.go (hunk 2 above)
Comment (oshoval) [reply tests/swap_test.go:227]: It is not related to the PR scope (appears on PR title and desc), 
lets focus please on the PR scope
adding it to another commit can be even confusing

File: tests/libnode/node.go (hunk 1 above)
Comment (oshoval) [reply tests/libnode/node.go]: Done

File: tests/swap_test.go (hunk 2 above)
Comment (codingben) [reply tests/swap_test.go]: It's up to you, but if you already adding new changes, I think you still can remove it from the function signature.

**Pull request merged**